src/overlay_params.cpp

```diff
@@ -660,7 +660,6 @@ parse_overlay_config(struct overlay_params *params,
    if (!env || read_cfg) {
 
       // Get config options
-      bool has_config_file = parseConfigFile(*params);
```

Comment:
This is half-send revert of https://github.com/flightlessmango/MangoHud/commit/beecde110b54badbb7ba4a8255c265a0e506d737
`parseConfigFile(*params);` should stay probably.

Reply:
Would be better to finalise the revert, wouldn't it? One can always cherry-pick the original patch as needed in the future.